.github/issue_template.md

```diff
@@ -21,6 +21,9 @@
   - [ ] MySQL
   - [ ] MSSQL
   - [ ] SQLite
+- Are you using Gitea behind CloudFlare?
```

Comment:
Maybe instead of question, a comment like `<!-- When using a CDN like Cloudflare, please remember that they can modify page responses. Please disable CDN fully and confirm your issue still happens. -->`

Reply:
might be, let me change it

Reply:
I moved the comment to the description section, because I think it's more likely to get noticed there.